src/selectors/dataTypes.ts

```diff
@@ -14,20 +14,25 @@ export const selectDataTypesByIds = (dataTypeIds: string[] | undefined) =>
     selectCurrentTaskId,
     selectInstanceData,
     (dataTypes = [], currentTask, instanceData) => {
-      const relevantDataTypes = dataTypes?.filter((type) => type.taskId === currentTask);
-      return instanceData?.filter((dataElement) => {
-        if (dataTypeIds) {
-          return dataTypeIds.findIndex((id) => dataElement.dataType === id) > -1;
-        }
-        return relevantDataTypes.findIndex((type) => dataElement.dataType === type.id) > -1;
-      });
+      const relevantDataTypes = dataTypes.filter((type) => type.taskId === currentTask);
+      const useSpecificDataTypeIds = dataTypeIds && !dataTypeIds?.includes('include-all');
```

Comment:
Since the strings `'ref-data-as-pdf'` and `'include-all'` etc, are strings that are used several places. Would it be beneficial to place them in an enum or a string-literal type mabye?